python/ctsm/test/test_sys_run_neon.py

```diff
@@ -58,7 +58,7 @@ def test_one_site(self):
 
         # assert that BART directories were created during setup
         self.assertTrue("BART" in glob.glob(self._tempdir + "/BART*")[0])
-        #self.assertTrue("BART" in glob.glob("BART*")[0])
+        # self.assertTrue("BART" in glob.glob("BART*")[0])
```

Comment:
Do we need to keep this alternate line around as a comment, or could we remove? I'm fine either way if there's justification.

Reply:
Ah, yes, I can remove that! Thanks for catching this.